com.unity.render-pipelines.universal/Runtime/ForwardLights.cs

```diff
@@ -377,7 +377,7 @@ public void Setup(ScriptableRenderContext context, ref RenderingData renderingDa
                 m_LightCookieManager.Setup(context, cmd, ref renderingData.lightData);
             }
             context.ExecuteCommandBuffer(cmd);
-            CommandBufferPool.Release(cmd);
```

Comment:
since I see the pattern that all the setup methods need the explicit ExecuteCommandBuffer and Clear at the end. Could we call this once from ScriptableRenderer, so we just flush the commands in between Setup and Execute, and every pass implementation doesn't have to do it manually every time?

Reply:
The way I understand there's no way to remove the executes from the passes until we move to RendererList, otherwise our pass profiler markers will all be broken. Even after moving everything to command buffer, I think it's still safe to execute in between passes like is done [here](https://github.com/Unity-Technologies/Graphics/pull/6877/files#diff-b727bc7307e94c250d91774218394ee2a30e47117dce2fa62f42d8948af31a63R462-R463) as we don't know if a user is injecting a custom pass and running his own command buffer as they have access to the ScriptableRenderContext / CommandBufferPool.